src/book/04-general/01-git.mdx

```diff
@@ -0,0 +1,59 @@
+---
+section: Guides
+chapter: General
+title: Git
+description: Contributing to a NUbots project with git.
+slug: /guides/general/git
+---
+
+## Prerequisites
+
+This guide assumes you know the basics of git and basic shell commands.
+
+## Getting Code Into a Repository
+
+Suppose you have a file you have made which you would like to add to a NUbots repository, called `my_file.cpp`, as part of your project to make the robot stand up.
```

Comment:
The wording is a little hard to follow. Maybe something like
```suggestion
Let's suppose you have made a file, which you would like to add to a NUbots repository, called `my_file.cpp`. Suppose this file is part of your project to make the robot stand up.
```

Reply:
I have made changes along these lines. let me know what u think.